frontend/src/components/MainComponent/MainComponent.tsx

```diff
@@ -64,7 +55,7 @@ function MainComponent({
 	// called on mount
 	useEffect(() => {
 		// perform backend health check
-		healthCheck().catch(() => {
+		healthService.healthCheck().catch(() => {
```

Comment:
This is interesting. Once you consolidate the service calls, I imagine you won't need this health check, because the MainComponent will mount at the same time as the "fetch level data" request is fired off.

Reply:
yet we need the health endpoint for the AWS load balancer, right?

Reply:
Correct. Seems we'd not need to call it from the frontend, but we'll still want the endpoint defined in the backend, for the load balancer.